examples/lox/src/language/type-system/lox-type-checking.ts

```diff
@@ -30,7 +30,7 @@ export function createTypir(domainNodeEntry: AstNode): Typir {
             return super.printDomainElement(domainElement, sentenceBegin);
         }
     }
-    typir.printer = new OxPrinter(typir);
+    typir.printer = new OxPrinter();
```

Comment:
I am surprised about this: I expected, that we have a custom LoxModule, which registers `OxPrinter` as printing service.

`typir.printer` should be read-only after initialization?